Refactor modules (#2217)
pyansys/pyaedt: doc/source/API/Primitives3D.rst

> Modeler and  primitives 2D/3D

Reviewer: ```suggestion
Modeler and primitives in 2D and 3D
```

> Modeler and  primitives 2D/3D
> =============================

Reviewer: ```suggestion
Modeler and primitives in 2D and 3D
=================================
```

> This section lists the core AEDT Modeler modules available in 2D and 3D Solver (HFSS, Maxwell, Icepak, Q3D, Mechanical):

Reviewer: ```suggestion
This section lists the core AEDT Modeler modules with both 2D and 3D solver (HFSS, Maxwell, Icepak,
Q3D, and Mechanical):
```

    # This call returns a CoordinateSystem object
    new_cs = app.modeler.create_coordinate_system()

    ...

Advanced modeler operations
~~~~~~~~~~~~~~~~~~~~~~~~~~~

> PyAEDT also includes some advanced modeler tool like ``MultiPartComponent`` for 3DComponent management
> and ``Stackup3D`` for 3D Modeler Stackup parametric creation.

Reviewer: ```suggestion
PyAEDT includes some advanced modeler tools like ``MultiPartComponent`` for 3D component management
and ``Stackup3D`` for parametric creation of 3D modeler stackups.
```
